display_list/display_list_builder.cc

```diff
@@ -1041,9 +1041,9 @@ void DisplayListBuilder::drawPicture(const sk_sp<SkPicture> picture,
                                      const SkMatrix* matrix,
                                      bool render_with_attributes) {
   matrix  //
-      ? Push<DrawSkPictureMatrixOp>(0, 1, std::move(picture), *matrix,
+      ? Push<DrawSkPictureMatrixOp>(0, 1, picture, *matrix,
                                     render_with_attributes)
-      : Push<DrawSkPictureOp>(0, 1, std::move(picture), render_with_attributes);
```

Comment:
This seems like a false alarm.  Only one move is guaranteed to execute and this technically changes the performance for the worse.

Reply:
The problem isn't the ternary, it's that we reference picture on the line right under this.

Reply:
Ahh missed that, nice catch. LGTM.

Reply:
(Well, the line under the comments under this, which GitHub is cutting off by default :)